<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remoteSyncDialog</class>
 <widget class="QWidget" name="remoteSyncDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Control Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>530</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>Remote Sync Plug-in Configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Remote Sync</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="serverGroupBox">
         <property name="title">
          <string>Server mode</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="serverButton">
            <property name="text">
             <string>Start server</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QWidget" name="serverControls" native="true">
            <layout class="QFormLayout" name="formLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Server port</string>
               </property>
               <property name="buddy">
                <cstring>serverPortSpinBox</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="serverPortSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>65535</number>
               </property>
               <property name="value">
                <number>20180</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_serverName">
            <property name="text">
             <string>Server Name, IP: </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>120</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="restoreDefaultsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Restore default settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveSettingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save settings as default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>StatusLabel</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QGroupBox" name="clientGroupBox">
         <property name="title">
          <string>Client mode</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="clientButton">
            <property name="text">
             <string>Connect to server</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QWidget" name="clientControls" native="true">
            <layout class="QFormLayout" name="formLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Server host address/name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="clientServerHostEdit">
               <property name="text">
                <string>127.0.0.1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Server port</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="clientServerPortSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>65535</number>
               </property>
               <property name="value">
                <number>20180</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string comment="tab in plugin windows">About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="aboutTextBrowser">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="openLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
